Override ds-branding defaults

xjensen opened the pull request:
We have another PR on the design-system repo, cagov/design-system#124, to modify some of the ds-branding component's default values. This PR aims to provide overrides for headless.cannabis.ca.gov, to ensure this site can continue to look correct alongside changes to official ds-branding.

We should wait to merge this until the ds-branding change is merged into design-system.

Let me know if any questions or concerns!

aaronhans reviewed (approved):
cool

kkoryaka commented:
@xjensen Maybe there is a way to do it without overriders? Why don't we fix what we need directly in the design system?

aaronhans commented:
We have color css variables, we could expand the variable set to include some spacing css vars, also some font ones so that overrides could be more organized

I think this update where we get to use the branding component on a new site is a great first step though

xjensen commented:
I agree with @kkoryaka's general idea here.

We would probably need 10+ new CSS vars to implement Cannabis/Drought within the design-system as-is. We might need to rewrite the component a bit instead. I'm thinking about that now.

For the future, I'm wondering what lives in design-system vs. on each site. Juggling three repos for this change was already a challenge, and it's only going to get worse as we take on more sites.

xjensen commented:
I've rewritten parts of the upstream ds-branding component. Those changes should help accommodate Drought and Cannabis without overriding CSS. It's a big change to the internal CSS; I still need to add additional documentation.

I've also added a commit here to demonstrate the changes that would be needed in Cannabis. The markup needs a few small edits. There are also a lot of new CSS variables to control font settings. No more CSS overrides. 

Please take a look! Let me know what you think. Hopefully this is a more supportable solution.

(One note: I did update the other CSS color vars here too, to match recent changes, just so I could get a good look at everything in testing. But the only real additions needed for this PR would be the font-related changes at the bottom of the cannabis.scss file.)

aaronhans commented on src/css/sass/colorschemes/cannabis.scss:
```diff
@@ -7,27 +7,30 @@
+  --highlight-lightest-color: #ECD0C4;
+  --secondary-hover-color: #864529;
+  
+  --org-name-state-font: "Arial", "Helvetica", sans-serif;
```
nice variable naming
these all make sense

aaronhans commented on src/css/sass/colorschemes/cannabis.scss:
```diff
@@ -7,27 +7,30 @@
-  --standout-color: #2f4c2c;
-  --standout-hover-color: #21351f; 
-  --logo-font: 'Lexend';
+  --secondary-color: #2F4C2C;
```
are these color changes intentional or are they coming through from another update?

xjensen replied:
Yep, I copied over the color changes from something else that Konstantin was doing for Cannabis, just to see correct colors in dev testing. I'll defer to Konstantin's changes when I handle the merge conflict, but looks like it's all the same.

xjensen replied:
I did change the name of the --logo-font variable to match all the other variables. I didn't see any other uses of this in Cannabis, but we should probably check with @kkoryaka just to be sure.

xjensen replied:
Resolved conflicts with main branch.

aaronhans commented:
Impressive you've gotten rid of the overrides! 

Please verify color change variable values are desired

xjensen commented:
Regarding color changes, Konstantin merged the official changes into `main` a day or two ago. I just merged his changes into this branch to ensure we're all synced up.
